{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env sh
 {% include 'misc/header.py' %}
 
-pydocstyle {{ cookiecutter.package_name }} && \
+pydocstyle {{ cookiecutter.package_name }} tests && \
 isort -rc -c -df **/*.py && \
-check-manifest --ignore ".travis-*" && \
+check-manifest --ignore ".travis-*,docs/_build" && \
```

Comment:
Then let's go then with travis ci agnostic way ... but can we do it in separate issue/PR so we can discuss it in wider circle (e.g. `@inveniosoftware/triagers or developers`).